locale/dbr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,8639 @@
+# , fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT 1.0\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-05-17 11:23+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"Generated-By: Babel 2.4.0\n"
+
+msgid "No links are allowed."
+msgstr ".pǝʍoʅʅɐ ǝɹɐ sʞuıʅ oN"
+
+msgid "This display name cannot be used."
+msgstr ".pǝsn ǝq ʇouuɐɔ ǝɯɐu ʎɐʅdsıp sıɥ⊥"
+
+msgid "Usernames cannot contain only digits."
+msgstr ".sʇıƃıp ʎʅuo uıɐʇuoɔ ʇouuɐɔ sǝɯɐuɹǝs∩"
+
+msgid "Enter a valid username consisting of letters, numbers, underscores or hyphens."
+msgstr ".suǝɥdʎɥ ɹo sǝɹoɔsɹǝpun ´sɹǝqɯnu ´sɹǝʇʇǝʅ ɟo ƃuıʇsısuoɔ ǝɯɐuɹǝsn pıʅɐʌ ɐ ɹǝʇuƎ"
+
+msgid "This username cannot be used."
+msgstr ".pǝsn ǝq ʇouuɐɔ ǝɯɐuɹǝsn sıɥ⊥"
+
+msgid "This username is already in use."
+msgstr ".ǝsn uı ʎpɐǝɹʅɐ sı ǝɯɐuɹǝsn sıɥ⊥"
+
+msgid "Images must be either PNG or JPG."
+msgstr ".⅁Ԁſ ɹo ⅁NԀ ɹǝɥʇıǝ ǝq ʇsnɯ sǝƃɐɯI"
+
+#, python-format
+msgid "Please use images smaller than %dMB."
+msgstr ".ԐW%d uɐɥʇ ɹǝʅʅɐɯs sǝƃɐɯı ǝsn ǝsɐǝʅԀ"
+
+msgid "You are already logged in."
+msgstr ".uı pǝƃƃoʅ ʎpɐǝɹʅɐ ǝɹɐ no⅄"
+
+msgid "Your login attempt could not be parsed. Please try again."
+msgstr ".uıɐƃɐ ʎɹʇ ǝsɐǝʅԀ .pǝsɹɐd ǝq ʇou pʅnoɔ ʇdɯǝʇʇɐ uıƃoʅ ɹno⅄"
+
+msgid "Your Firefox Account could not be found. Please try again."
+msgstr ".uıɐƃɐ ʎɹʇ ǝsɐǝʅԀ .punoɟ ǝq ʇou pʅnoɔ ʇunoɔɔ∀ xoɟǝɹıℲ ɹno⅄"
+
+msgid "You could not be logged in. Please try again."
+msgstr ".uıɐƃɐ ʎɹʇ ǝsɐǝʅԀ .uı pǝƃƃoʅ ǝq ʇou pʅnoɔ no⅄"
+
+msgid "Need help?"
+msgstr "¿dʅǝɥ pǝǝN"
+
+msgid "Review"
+msgstr "ʍǝıʌǝᴚ"
+
+msgid "Version {0}"
+msgstr "{0} uoısɹǝɅ"
+
+msgid "Review note"
+msgstr "ǝʇou ʍǝıʌǝᴚ"
+
+msgid "Only latest versions of addons can have notes added."
+msgstr ".pǝppɐ sǝʇou ǝʌɐɥ uɐɔ suoppɐ ɟo suoısɹǝʌ ʇsǝʇɐʅ ʎʅuO"
+
+msgid "Download Now"
+msgstr "ʍoN pɐoʅuʍoᗡ"
+
+msgid "Download"
+msgstr "pɐoʅuʍoᗡ"
+
+#. L10n: please keep &nbsp; in the string so &rarr; does not wrap.
+msgid "Continue to Download&nbsp;&rarr;"
```

Comment:
Need to handle this in potools. It's not the end of the world but would be good to make it ignore HTML entities.